libs/libsearpc/Makefile

```diff
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libsearpc
-PKG_VERSION:=5.1.4
+PKG_VERSION:=6.0.7
```

Comment:
You change here the version for libsearpc, but do not change the git commit hash to be fetched.
Intentionally?